app/controllers/api/migration_analytics_manifest_controller.rb

```diff
@@ -0,0 +1,34 @@
+def parse_json_if_valid(json)
+    return JSON.parse(json)
```

Comment:
No need for the explicit return, you can just do `JSON.parse(json)`

Reply:
Good to know.